engli/108_visloge.md

```diff
@@ -6,26 +6,24 @@ Pronouns can substitute nouns and noun phrases.
 
 | Singular    | Plural       |
 |:------------|:-------------|
-| **me**      | **mome**     |
+| **mo**      | **momen**    |
```

Comment:
-men works in the 2nd and 3rd person, but not so well in the 1st – "we" isn't the plural of "I".

Reply:
I don't see any real problem in there. In a strict sense you are right but pronouns are seldom used so strictly. I/we means the person/people I speak for, myself or my people. You/you all means the person/people I speak to, yourself or your people. He/she/they means the person/people I speak about.

It's possible extend plurals to people that are not really involved by saying things like "We the Finns admire you Germans" even if in reality there is only one Finn saying it and only one German hearing it. So the plural pronouns can also represent others than only the speaker the the listener(s).